README.md

```diff
@@ -5,3 +5,9 @@
 This is a simple python interface of RARBG.to.
 
 The api object will automatically fetch/refresh the token and take care rate control (1 request/2 seconds)
+
+# using
+
+    pip install RarbgAPI
+
+For an example, see matrix.py
```

Comment:
And we can have a  `quickstart` session to show a way to go instead of submit a new file.

Reply:
I don't understand. Can you elaborate?